packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -539,6 +539,7 @@ const TRANSFORMS = {
 
       allocationStrategy: ALLOCATION_BY_TYPE[srType],
       current_operations: obj.current_operations,
+      in_maintenance_mode: obj.other_config['xo:maintenanceState'] !== undefined,
```

Comment:
- camelCase
- `enabledMaintenanceMode`?

Reply:
Or simply `disabled`, like the hosts. /cc @julien-f

Reply:
I'm not fond of `disabled`, I don't know what it means, whereas `host.enabled` is defined in the XAPI.

`inMaintenanceMode` looks good to me.